queries/buildings.jinja2

```diff
@@ -57,10 +58,11 @@ SELECT
     NULL AS area,
     osm_id AS __id__,
     NULL AS "building:part",
+    "address" AS kind,
```

Comment:
You'll want `'address'` instead of `"address"` if it's supposed to be a string. PostgreSQL uses double quotes for column and table names, single quotes for strings.